core/services/ocr2/plugins/mercury/config/config.go

```diff
@@ -23,11 +26,14 @@ type PluginConfig struct {
 	// server does not have any previous reports. For a brand new feed, this
 	// effectively sets the "first" validFromBlockNumber.
 	InitialBlockNumber null.Int64 `json:"initialBlockNumber" toml:"initialBlockNumber"`
+
+	LinkFeedID   *common.Hash `json:"linkFeedID" toml:"linkFeedID"`
+	NativeFeedID *common.Hash `json:"nativeFeedID" toml:"nativeFeedID"`
```

Comment:
Is the code at this level meant to be chain-agnostic?

Reply:
The report is consumable on EVM-compatible chains, native fee is in eth

Reply:
What I mean is, should the EVM specific details be exposed here at all? this code was previously chain-agnostic - was that by design? This separation is important for LOOP Plugins.

Reply:
Mm, although mercury v0.3 is an evm-only solution for now, I can't say those are specific EVM details. We need the feedIds to fetch the corresponding reports from Mercury server and extract actual prices from there to calculate the report verification fee. Technically, it's just a key. Would it help if it becomes `[32]byte` instead of `common.Hash`?

Reply:
also cc @samsondav

Reply:
The typical approach is to use a string as long as you don't need to parse it. Only the evm relayer should be parsing or interpreting it.
https://github.com/smartcontractkit/libocr/blob/master/offchainreporting2plus/types/types.go#L297-L298